src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { AppRoutes, ServerApp } from './presentation'
+
+import { envs } from './configuration'
+import { MongoDatabase } from './data'
+;(() => {
+  main()
+})()
+
+async function main() {
+  await MongoDatabase.connect({
+    dbName: envs.MONGO_BD_NAME,
+    mongoUrl: envs.MONGO_URL,
+  })
+  new ServerApp({ port: Number(envs.PORT), router: AppRoutes.routes }).start()
```

Comment:
si ya **envs.PORT** no hace falta que lo castees con una clase Number
```suggestion
  new ServerApp({ port: envs.PORT, router: AppRoutes.routes }).start()
```